functions.php

```diff
@@ -21,17 +21,27 @@ function moh_scripts_styles() {
 
 	wp_register_style( 'global.css', get_stylesheet_directory_uri().'/css/build/minified/global.css', array( 'libraries-global' ), '20140423' );
 
+	wp_register_style( 'bootstrapCSS', get_stylesheet_directory_uri() . '/css/bootstrap.css', 'false', '', false );
+
+
+
 	wp_register_script( 'easyXDM', get_stylesheet_directory_uri().'/js/libs/easyXDM.min.js', array( 'jquery' ), false, false );
 
 	wp_register_script( '3play', get_stylesheet_directory_uri().'/js/libs/3playmedia.js', array( 'jquery' ), '3.0', true, true );
 	wp_register_script( '3play-player', get_stylesheet_directory_uri().'/js/libs/3play.player.js', array( 'jquery' ), false, true );
 	wp_register_script( '3play-hack', get_stylesheet_directory_uri().'/js/3play.hack.js', array( 'jquery', '3play', '3play-player' ), false, true );
 
+	wp_register_script( 'bootstrap-js', '//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js', array( 'jquery' ), true ); // All the bootstrap javascript goodness.
+
+
+
 	/* Queue scripts and styles */
 
 	wp_enqueue_style( 'global.css' );
 	wp_enqueue_style( 'twentytwelve-style', get_stylesheet_uri(), array(), '20140328' );
-
+	
+	wp_enqueue_style( 'bootstrap', '//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css' );
```

Comment:
If we have to add bootstrap, these lines (43-44) should come before the theme styles so the theme styles will override the bootstrap ones.

Reply:
(The bootstrap styles are causing a lot of the visual variations including the footer one that @matt-bernhardt mentioned.)

Reply:
Thanks, I have reorganized the styles